components/Form/Checkboxes/StyledCheckbox/StyledCheckbox.tsx

```diff
@@ -23,23 +23,43 @@ const StyledCheckbox: React.FC<PropTypes> = ({
     style,
     ...props
 }) => (
-    <label
-        className={clsx(
-            styles.checkboxLabel,
-            checked && styles.checked,
-            className
-        )}
-        style={style}
-    >
-        <input
-            type="checkbox"
-            className={styles.checkbox}
-            value={value}
-            checked={checked}
-            {...props}
-        />
-        {label}
-    </label>
+    <div className={styles.checkboxWrapper}>
+        <label
+            className={clsx(
+                styles.checkboxLabel,
+                checked && styles.checked,
+                className
+            )}
+            style={style}
+        >
+            <input
+                type="checkbox"
+                className={styles.checkbox}
+                value={value}
+                checked={checked}
+                {...props}
+            />
+            <svg
```

Comment:
Put this svg under `/public/form/Checkbox.svg` - avoid directly embedding svgs so caching optimizations & such don't rerender this every time.